### ISSUE TITLE
Can't resolve fs,os and path 

### ISSUE THREAD

**Issue by koernchen02** [permafrost-dev/node-ray#2]
Please close this if you find that _report_ to unmotivated, respectively it is lacking important details , just wanted to let you know!
Otherwise I'm happy to provide more details.

**Describe the bug**

Can not build my dist files when using laravel-mix:6.0.10 which uses webpack:5.1.3.

**Environment (please complete the following information):**

 - OS: Ubuntu 20.04 (Docker Environment)
 - NodeJS Version tried v14.15.4 & 15.8.0

**Additional context**
The error message I get when trying to run mixes `watch` or `dev` command:

```
> @ watch /var/www/html
> mix watch

WARNING in ./node_modules/node-ray/dist/index.cjs 1215:22-45
Critical dependency: the request of a dependency is an expression

ERROR in ./node_modules/node-ray/dist/index.cjs 8:9-22
Module not found: Error: Can't resolve 'fs' in '/var/www/html/node_modules/node-ray/dist'

ERROR in ./node_modules/node-ray/dist/index.cjs 9:9-22
Module not found: Error: Can't resolve 'os' in '/var/www/html/node_modules/node-ray/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
	- install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "os": false }

ERROR in ./node_modules/node-ray/dist/index.cjs 10:11-26
Module not found: Error: Can't resolve 'path' in '/var/www/html/node_modules/node-ray/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }

ERROR in ./node_modules/node-ray/node_modules/find-up/index.js 2:13-28
Module not found: Error: Can't resolve 'path' in '/var/www/html/node_modules/node-ray/node_modules/find-up'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }

ERROR in ./node_modules/node-ray/node_modules/locate-path/index.js 2:13-28
Module not found: Error: Can't resolve 'path' in '/var/www/html/node_modules/node-ray/node_modules/locate-path'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }

ERROR in ./node_modules/node-ray/node_modules/locate-path/index.js 3:11-24
Module not found: Error: Can't resolve 'fs' in '/var/www/html/node_modules/node-ray/node_modules/locate-path'

ERROR in ./node_modules/node-ray/node_modules/path-exists/index.js 2:11-24
Module not found: Error: Can't resolve 'fs' in '/var/www/html/node_modules/node-ray/node_modules/path-exists'

webpack compiled with 7 errors and 1 warning

```

**Comment by patinthehat** [comment]
Please update to the latest version of the package _(v1.2.1 as of now)_, and try using the web build:
```js
import { ray } from 'node-ray/web';
const { ray } = require('node-ray/web').ray;
```
or if you're referencing the file directly, it's located at `node_modules/node-ray/dist/web.cjs`.

Hopefully this will resolve the issue for you.

**Comment by koernchen02** [comment]
Now I have only one error left:

```
yarn watch
yarn run v1.22.5
$ mix watch
ERROR in ./node_modules/node-ray/dist/web.cjs 9:9-22
Module not found: Error: Can't resolve 'os' in '/var/www/html/node_modules/node-ray/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
	- install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "os": false }

webpack compiled with 1 error

```
I'm so sorry, only posting the error messages here!

**Comment by patinthehat** [comment]
I realized that there was an `'os'` import that I missed.  I just published version 1.2.2, which should resolve this last issue for you. Please let me know if you run into any other issues.

Thank you!
